internal/producer/src/accounts/accounts.go

```diff
@@ -392,3 +340,98 @@ func ValidateContract(c *Contract, table string, authorizedMinters [][]byte) (bo
 
 	return false, errors.New("Failed to validate contract")
 }
+
+type AccountInfo struct {
+	balance    uint64
+	stateNonce uint64
+}
+
+func NewAccountInfo(balance uint64, stateNonce uint64) *AccountInfo {
+	return &AccountInfo{balance: balance, stateNonce: stateNonce}
+}
+
+func (accInfo *AccountInfo) Serialize() ([]byte, error) {
+	serializedAccount := make([]byte, 16) // 8 + 8 bytes for balance and stateNonce
+	binary.LittleEndian.PutUint64(serializedAccount[:8], accInfo.balance)
+	binary.LittleEndian.PutUint64(serializedAccount[8:], accInfo.stateNonce)
+	return serializedAccount, nil
+}
+
+func (accInfo *AccountInfo) Deserialize(serializedAccountInfo []byte) error {
+	// accInfo = &AccountInfo{
+	// 	balance:    binary.LittleEndian.Uint64(serializedAccountInfo[:8]),
+	// 	stateNonce: binary.LittleEndian.Uint64(serializedAccountInfo[8:])}
```

Comment:
What is the purpose of these comments?

Reply:
Oh, I forgot to remove these comments before committing.. Sorry about that